common/views/components/LoadingIndicator/LoadingIndicator.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import { useEffect } from 'react';
+import NProgress from 'nprogress';
+import Router from 'next/router';
+import styled from 'styled-components';
+
+const LoadingIndicatorWrapper = styled.div.attrs({
+  className: 'loading-indicator-wrapper',
+})`
+  #nprogress {
+    pointer-events: none;
+  }
+  #nprogress .bar {
+    background: ${props => props.theme.colors.yellow};
+    position: fixed;
+    z-index: 1031;
```

Comment:
I hope I don't have to tell you I didn't author these styles myself – they're the [nprogress defaults](https://unpkg.com/nprogress@0.2.0/nprogress.css) copy-pasta